bin/oref0-radio-tune.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+source $(dirname $0)/oref0-bash-common-functions.sh || (echo "ERROR: Failed to run oref0-bash-common-functions.sh. Is oref0 correctly installed?"; exit 1)
+
+usage "$@" <<EOF
+Usage: $self
+Determine the optimum frequency to communicate with the pump and write
+the output to monitor/mmtune.json and monitor/medtronic_frequency.ini
+EOF
+
+export MEDTRONIC_PUMP_ID=`grep serial pump.ini | tr -cd 0-9`
+OREF0_DEBUG=${OREF0_DEBUG:-0}
+if [[ "$OREF0_DEBUG" -ge 1 ]] ; then
+  exec 3>&1
+else
+  exec 3>/dev/null
+fi
+if [[ "$OREF0_DEBUG" -ge 2 ]] ; then
+  exec 4>&1
+  set -x
+else
+  exec 4>/dev/null
+fi
+
+function mmtune_Go() {
+  set -o pipefail
+  if ( grep "WW" pump.ini ); then
+    Go-mmtune -ww | tee monitor/mmtune.json
+  else
+    Go-mmtune | tee monitor/mmtune.json
+  fi
+}
+
+echo {} > monitor/mmtune.json
+echo -n "mmtune: " && mmtune_Go >&3 2>&3
+# if mmtune.json is empty, re-run it and display output
+if ! [ -s monitor/mmtune.json ]; then
+    mmtune_Go
+fi
+#Read and zero pad best frequency from mmtune, and store/set it so Go commands can use it,
+#but only if it's not the default frequency
+if ! $([ -s monitor/mmtune.json ] && jq -e .usedDefault monitor/mmtune.json); then
+  freq=`jq -e .setFreq monitor/mmtune.json | tr -d "."`
+  while [ ${#freq} -ne 9 ];
+    do
+     freq=$freq"0"
+    done
+  #Make sure we don't zero out the medtronic frequency. It will break everything.
+  if [ $freq != "000000000" ] ; then
+       echo $freq > monitor/medtronic_frequency.ini
+  fi
```

Comment:
perhaps also `echo $freq` to the user. And make to not end with a `echo -n`. 
Otherwise output from CLI will look like
```
# oref0-radio-tune
mmtune: root@edisonrig:~/myopenaps#
```